hawc_hal/healpix_handling/sparse_healpix.py

```diff
@@ -100,6 +131,10 @@ def set_new_values(self, new_values):
 
         self._partial_map[:] = new_values
 
+    @property
+    def pixels(self):
```

Comment:
I would call this `pixel_ids`, to make clearer that these are the Healpix IDs of the pixels, not their values